pants.travis-ci.toml

```diff
@@ -11,5 +11,13 @@ pantsd = false
 [test]
 use_coverage = true
 
+[pytest]
+junit_xml_dir = "dist/test-results/"
+
 [coverage-py]
 report = ["raw", "xml"]
+
+[auth]
+from_env_var = "TOOLCHAIN_AUTH_TOKEN"
+customer = "pantsbuild"
```

Comment:
Nit: renaming this to `user` or `org` would be more neutral.

Reply:
this requires a new version of the plugin, I will be addressing that later on.